test/e2e/windows_exporter_test.go

```diff
@@ -0,0 +1,43 @@
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	core "k8s.io/api/core/v1"
+)
+
+// testWindowsExporter deploys Linux pod and tests that it can communicate with Windows node's metrics port
+func testWindowsExporter(t *testing.T) {
```

Comment:
How is this different from the `testMetrics` test above?

Reply:
I have the same question.

Reply:
We decided to rename `testMetrics` to `testWindowsExporter`, reasoning here https://github.com/openshift/windows-machine-config-operator/pull/163#discussion_r518025558